include/envoy/upstream/outlier_detection.h

```diff
@@ -129,9 +130,9 @@ enum class EjectionType { Consecutive5xx, SuccessRate, ConsecutiveGatewayFailure
 /**
  * Sink for outlier detection event logs.
  */
-class EventLogger {
+class DetectionEventLogger {
```

Comment:
sorry, but I still don't think this merits a name change. I am open to be outvoted. @jmarantz do you have an opinion one way or another?

Reply:
@junr03 was mainly to keep consistency of event loggers (currently only 2) naming pattern across project. No strong opinion on that, will rename back.